i18n/api/en.ts

```diff
@@ -46,7 +46,8 @@ const en: Translations = {
       'When would you like to start receiving the Old Age Security (OAS) pension?',
     [FieldKey.OAS_AGE]:
       'At what age would you like to start receiving the OAS pension?',
-    [FieldKey.MARITAL_STATUS]: 'What is your marital status?',
+    [FieldKey.MARITAL_STATUS]: 'What is your current marital status?',
```

Comment:
Could you remove the "current"? I was asked to remove the "current" in another task. Otherwise, it looks good to me.